source/validate.cpp

```diff
@@ -286,6 +291,12 @@ spv_result_t spvValidateIDs(const spv_instruction_t* pInsts,
 }
 
 namespace {
+// Shame
```

Comment:
:)

Instead of `CHECK_RESULT(expr)`, why can't you use, eg, `if (auto error = expr) return error;` ??